killbill_client.gemspec

```diff
@@ -39,8 +39,6 @@ Gem::Specification.new do |s|
 
   s.rdoc_options << '--exclude' << '.'
 
-  s.add_dependency 'json', '>= 1.2.0', '< 2.0.0'
```

Comment:
I don't think we can fully remove it unfortunately, it looks like we need it at least for ree-1.8.7-2011.12 (which we still need to support 🙄): https://travis-ci.org/killbill/killbill-client-ruby/builds/330403156

Maybe we can only add it as a dependency for old versions of rubies?